### PR TITLE
Filter policy updated

### PR DESCRIPTION
**Pull request by vbagirov**
+ Annoyances filter
+ Useful ads filter
+ Other minor changes

### CODE REVIEW

**Comment by vbagirov** [comment]
Should we also include the Mobile Ads filter in the policy?

File: 02.general/05.adguard-filter-policy/docs.en.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ When discussing AdGuard ad filters, there is often one topic that causes dispute
 
 ## Ad filters
 
-Those are actually language filters, including: Russian, English, German, Dutch, Spanish, Portuguese, Japanese.
+This part describes AdGuard Base filter, Mobile ads filter and following language-specific filters: Russian, German, Dutch, Spanish/Portuguese, Japanese, Turkish, French.
 
 ##### What shall these filters block?
 
-- The filters shall block ads everywhere possible. 
+- These filters shall block ads everywhere possible.
```
Comment (TheHasagi) [inline 02.general/05.adguard-filter-policy/docs.en.md]: These filters should block ads wherever possible. - Звучит лучше

File: 02.general/05.adguard-filter-policy/docs.en.md [hunk 2]
```diff
@@ -69,8 +69,42 @@ If a rule is subject to the list of limitations described below, then it won’t
...
+- Any elements that are essential for the websites' functioning (e.g. authorization forms) will not be blocked even if they satisfy other requirements.
+- Rules that cause problems with functioning of websites will be removed.
+- Rules that are specific to the particular website will be added only if the website has sufficient traffic.  
+ Traffic is determined by open statistics (if it has such) or in other ways.  
+ Website’s traffic is considered sufficient when it has 30 thousands visitors a month.
+
+
+## Filter for useful ads
+
+Unlike other filters, this filter **unblockds** certain ads. More about it in [this Knowledge Base article](https://kb.adguard.com/en/general/search-ads-and-self-promotion).
```
Comment (TheHasagi) [inline 02.general/05.adguard-filter-policy/docs.en.md]: **unblocks**

File: 02.general/05.adguard-filter-policy/docs.ru.md [hunk 3]
```diff
@@ -69,10 +70,44 @@ visible: true
...
+- Любые элементы, необходимые для нормального функционирования сайта (например, формы авторизации), не будут заблокированы, даже если они подходят под определение.
+- Правила, которые часто вызывают проблемы с работой каких-либо сайтов, будут удаляться.
+- Правила, специфичные для конкретного сайта, будут добавляться, только если сайт обладает достаточной посещаемостью.  
+ Посещаемость определяется по открытой статистике (если таковая присутствует), либо по косвенным признакам.  
+ Достаточной считается посещаемость от 30 тысяч посетителей в месяц.
+
+
+## Фильтр полезной рекламы
+
+В отличие от других фильтров, этот фильтр **раблокирует** определённую рекламу. Узнать об этом больше можно в специальной [статье в Базе Знаний](https://kb.adguard.com/ru/general/search-ads-and-self-promotion).
```
Comment (TheHasagi) [inline 02.general/05.adguard-filter-policy/docs.ru.md]: **разблокирует**

File: 02.general/05.adguard-filter-policy/docs.en.md [hunk 4]
```diff
@@ -69,8 +69,42 @@ If a rule is subject to the list of limitations described below, then it won’t
...
+If a rule is subject to the list of limitations described below, then it won’t be added to this filter.
+
+- Any elements that are essential for the websites' functioning (e.g. authorization forms) will not be blocked even if they satisfy other requirements.
+- Rules that cause problems with functioning of websites will be removed.
+- Rules that are specific to the particular website will be added only if the website has sufficient traffic.  
+ Traffic is determined by open statistics (if it has such) or in other ways.  
+ Website’s traffic is considered sufficient when it has 30 thousands visitors a month.
+
+
+## Filter for useful ads
```
Comment (ameshkov) [inline 02.general/05.adguard-filter-policy/docs.en.md]: It has a different name now

**Pull request merged**